[2023-01-17T22:46:32.585+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.store_user scheduled__2023-01-16T00:00:00+00:00 [queued]>
[2023-01-17T22:46:32.603+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.store_user scheduled__2023-01-16T00:00:00+00:00 [queued]>
[2023-01-17T22:46:32.603+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-01-17T22:46:32.603+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-01-17T22:46:32.603+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-01-17T22:46:32.628+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): store_user> on 2023-01-16 00:00:00+00:00
[2023-01-17T22:46:32.640+0000] {standard_task_runner.py:55} INFO - Started process 181 to run task
[2023-01-17T22:46:32.651+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'store_user', 'scheduled__2023-01-16T00:00:00+00:00', '--job-id', '191', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpvxhhc38b']
[2023-01-17T22:46:32.657+0000] {standard_task_runner.py:83} INFO - Job 191: Subtask store_user
[2023-01-17T22:46:32.781+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.store_user scheduled__2023-01-16T00:00:00+00:00 [running]> on host 274e1da73cdf
[2023-01-17T22:46:32.906+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2023-01-16T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-01-16T00:00:00+00:00
[2023-01-17T22:46:32.908+0000] {postgres.py:129} INFO - Running copy expert: COPY users FROM stdin WITH DELIMITER as ',', filename: /tmp/processed_user.csv
[2023-01-17T22:46:32.924+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-01-17T22:46:32.934+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-01-17T22:46:32.955+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=store_user, execution_date=20230116T000000, start_date=20230117T224632, end_date=20230117T224632
[2023-01-17T22:46:32.988+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-01-17T22:46:33.015+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
